ibis/backends/dask/dispatch.py

```diff
@@ -0,0 +1,11 @@
+from ibis.backends.pandas.dispatch import (  # noqa: F401
+    execute,
+    execute_literal,
```

Comment:
How does this work? These pandas dispatch methods are defined to take and return pandas Series. It looks like you are reusing this to return dd.Series? How does that work?

Reply:
Can you explain, perhaps, how does this work for Dask backend?

https://github.com/ibis-project/ibis/blob/master/ibis/backends/pandas/execution/selection.py#L372

I believe you dispatch into this but this method has code like

```
    # create a sequence of columns that we need to drop
    temporary_columns = pd.Index(
        concatv(grouping_keys, ordering_keys)
    ).difference(data.columns)

    # no reason to call drop if we don't need to
    if temporary_columns.empty:
        return result

    # drop every temporary column we created for ordering or grouping
    return result.drop(temporary_columns, axis=1)
```
How does this code work for dask Series as is?

Reply:
As requested, I haven't implemented anything inside `execution` for the dask backend yet (that's part 3 of https://github.com/ibis-project/ibis/issues/2537). You can see what the `execution` pieces would look like in https://github.com/ibis-project/ibis/pull/2400. We use the pandas machinery, but files under `execution` will register dask types where appropriate.

Reply:
Ok cool. I wasn't sure what is working and what is not working yet. 

Sounds like no node execution rule (e.g., rule for Selection) is implemented in this PR yet?

Reply:
Yep - exactly. I had implemented it in https://github.com/ibis-project/ibis/pull/2400, but 11k lines is too big of a PR to actually review so based on Jeff's suggestion I'm splitting this into 4-6 smaller PRs. Part 1 was the skeleton, Part 2 (this PR) borrows the pandas machinery to build up the dask client, and Part 3 (upcoming) will actually deal with the `execution` components.

Reply:
Got it. Sounds good to me.